cli/build.gradle.kts

```diff
@@ -0,0 +1,103 @@
+plugins {
+    `java-library`
+    application
+    id("org.graalvm.buildtools.native") version "0.10.3"
+}
+
+dependencies {
+    implementation("info.picocli:picocli:4.7.6")
+    annotationProcessor("info.picocli:picocli-codegen:4.7.6")
+
+    implementation("software.amazon.smithy:smithy-aws-traits:1.56.0")
```

Comment:
Use `implementation(libs.smithy.aws.traits)` and the like for the maven-sourced dependencies. This sources the version from [libs.version.toml](https://github.com/smithy-lang/smithy-java/blob/main/gradle/libs.versions.toml) so we have a single consistent set of dependencies for all modules. If you're editing this in an IDE, you should be able to get these with autocomplete.